### DISCUSSION TITLE
PHP setting for PHP version 7.3 on Ubuntu

### DISCUSSION THREAD

**Discussion by pa2eon** [magicbug/Cloudlog#1396] [General]
Dears,

At the ubuntu server (20.04.x)  I can think that a few PHP.ini settings must be optimised ?
Because the waiting time to collect the QTH information at time filling in the Live QSO,  takes to long.
I haver no errors in ther Apache log or elsewhere.

Any idea to fix this ?

Regards, Eric / PA2EON

**Comment by pa2eon** [comment]
Update - I had checked the DNS settings of the Linux server .. their was an IP number added who was not good.
Removed this .. and the request to the QRZ.com site is full up and running ..
Now checking on the radio .. keep informed.
Eric

**Comment by magicbug** [comment]
Could just be that the "host" is very slow at executing PHP scripts but there's not a huge amount I can do to make it faster.

**Comment by pa2eon** [comment]
Hi Peter, was not the problem .. I have found an ip number in the dns
settings who was wrong .. so the host did not get a good result of the ip
request from and to internet.  vy 73 , PA2EON

Op zo 6 feb. 2022 om 22:45 schreef Peter Goodhall ***@***.***>

> Could just be that the "host" is very slow at executing PHP scripts but
> there's not a huge amount I can do to make it faster.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/magicbug/Cloudlog/discussions/1396#discussioncomment-2121650>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABEAKLN3XOMB6EB5T4AAOALUZ3TX3ANCNFSM5NUDWARA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 

Met vriendelijke groeten,
Best regards,

Eric Oosterbaan

Bramerslandstraat 7a
5275 HE Den Dungen
T: +31 73 5530052

E: ***@***.***
W: www.oosterbaan.eu
